unattended_installer/install_functions/checks.sh

```diff
@@ -311,7 +311,7 @@ function checks_ports() {
     ports=("$@")
 
     if command -v ss > /dev/null; then
-        port_command="ss -lntup | grep -q "
+        port_command="ss -lntup sport "
```

Comment:
In some of the tests you did in issue https://github.com/wazuh/wazuh-packages/issues/1857 you used `:` before each port number, ¿what does it change, is it irrelevant?

Reply:
Great observation.

![image](https://user-images.githubusercontent.com/72193239/203387027-9cb4e913-4391-4b55-b109-51c58268a22f.png)

This screenshot shows that the character `:` does not affect the output of the command.